opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -95,8 +87,9 @@ where
         self
     }
 
-    /// Configures the time a [PeriodicReader] waits for an export to complete
-    /// before canceling it.
+    /// Configures the timeout for an export to complete. PeriodicReader itself
+    /// does not enforce timeouts. Instead timeout is passed on to the exporter
```

Comment:
nit: `enforce timeout`, or there could be multiple timeout values?

Reply:
fixed.